src/icon-tab-bar.scss

```diff
@@ -453,6 +481,8 @@ $fd-icon-tab-bar-semantic-values: (
         transform: rotate(180deg);
       }
     }
+
+    width: 2rem;
```

Comment:
Where did you take this from?

Reply:
I think I added it, when was implementing overflow directive in ngx, for some version of directive and forgot to delete. 
Removed and checked ngx, works well without these styles.